### DISCUSSION TITLE
React Native tKey integration

### DISCUSSION THREAD

**Discussion by zolottareva** [Web3Auth/Web3Auth#1258] [Help]
When trying to implement custom auth in React Native (bare workflow) using **@toruslabs/customauth-react-native-sdk**, safari page with "unable to find this page" message opens.
Initialisation: 
```
            CustomAuth.init({
                redirectUri: deepLink,
                network: "testnet",
                enableLogging: true
            });
```

Call:
```
        const loginDetails = await CustomAuth.triggerLogin({
            name: 'Cognito',
            typeOfLogin: "jwt",
            verifier,
            clientId,
            jwtParams:{
                domain,
                verifierIdField: "email",
                id_token: awsToken
            }
        });
```

**Comment by YZhenY** [comment]
@grvgoel81 could you assist here

**Reply by grvgoel81** [reply]
@zolottareva Pls share screenshot and console error logs.

**Reply by zolottareva** [reply]
@grvgoel81 we've changed our configuration of Web3Auth dashboard, so I can't reproduce error with browser opening.
Now I'm just getting error in **triggerLogin** function. Here is log from catch expression:
`[Error: triggerLogin: ]`
